src/Apps/Order2/Routes/Details/Components/Order2DetailsHeader.tsx

```diff
@@ -10,12 +10,12 @@ export const Order2DetailsHeader = ({ order }: Order2DetailsHeaderProps) => {
   const orderData = useFragment(FRAGMENT, order)
 
   return (
-    <>
+    <Box backgroundColor="mono0" p={2}>
```

Comment:
This part about adding background to each element is to prepare for responsive stuff. As according to designs page will have background color of light grey and each component will have it's own while bg.